package.json

```diff
@@ -110,6 +110,7 @@
     "@silexlabs/grapesjs-storage-rate-limit": "^1.0.8",
     "@silexlabs/grapesjs-symbols": "1.0.40",
     "@silexlabs/grapesjs-ui-suggest-classes": "1.0.23",
+    "@silexlabs/grapesjs-keymaps-dialog": "^1.0.0",
```

Comment:
We need to publish the plugin on NPM.

Reply:
please update the version, it is now v1.0.15
https://www.npmjs.com/package/@silexlabs/grapesjs-keymaps-dialog